web/src/common/ModeLink.svelte

```diff
@@ -5,14 +5,9 @@
   import { pageTitle } from "./navbar";
 
   export let mode: Mode;
-  export let afterLink: (() => void) | undefined = undefined;
```

Comment:
Noticed this was unused

Reply:
Whoops! Some vestige of a bad idea of mine, no doubt.